pkg/audit/manager.go

```diff
@@ -481,7 +481,7 @@ func (am *Manager) auditFromCache(ctx context.Context) ([]Result, []error) {
 		}
 		resp, err := am.opa.Review(ctx, au)
 		if err != nil {
-			errs = append(errs, fmt.Errorf("validating %v %s/%s: %v", obj.GroupVersionKind().String(), obj.GetNamespace(), obj.GetName(), err))
+			am.log.Error(err, "Unable to review object from audit cache %v %s/%s", obj.GroupVersionKind().String(), obj.GetNamespace(), obj.GetName())
```

Comment:
```suggestion
			am.log.Error(err, "Unable to review object from opa cache %v %s/%s", obj.GroupVersionKind().String(), obj.GetNamespace(), obj.GetName())
```

Reply:
Looks like we are consistently using the upper case for error logs. e.g. output
> {"level":"error","ts":1664218151.4940047,"logger":"controller","msg":"Unable to look up object namespace","process":"audit","audit_id":

Reply:
updated to `opa cache` in couple of places